go.mod

```diff
@@ -50,11 +49,10 @@ require (
 	go.uber.org/automaxprocs v1.5.1
 	go.uber.org/goleak v1.1.12
 	golang.org/x/sys v0.0.0-20220422013727-9388b58f7150
-	google.golang.org/genproto v0.0.0-20220217155828-d576998c0009 // indirect
 	google.golang.org/grpc v1.46.0
 	google.golang.org/protobuf v1.28.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 // Make sure Prometheus version is pinned as Prometheus semver does not include Go APIs.
-replace github.com/prometheus/prometheus => github.com/prometheus/prometheus v1.8.2-0.20220117154355-4855a0c067e2
```

Comment:
Do you think we can update to latest prometheus and not use git hash anymore?

Reply:
If you mean `@latest`, I think we should stick to git hash (monthly wise). Otherwise, I have seen incompatibility with `@latest` mod version, which is not good, as anyone who updates the mod next might have to fix the incompatibilities.